openverse_catalog/dags/common/dag_factory.py

```diff
@@ -77,12 +85,10 @@ def create_provider_api_workflow(
     dagrun_timeout:   datetime.timedelta giving the total amount of time
                       a given dagrun may take.
     """
-    args = deepcopy(default_args)
-    args.update(start_date=start_date)
-    print(args)
+    default_args = default_args or DAG_DEFAULT_ARGS
     dag = DAG(
         dag_id=dag_id,
-        default_args=args,
+        default_args={**default_args, "start_date": start_date},
```

Comment:
Would this always set the start date to the default for `create_provider_api_workflow` `(1970, 1,1)`? Not that it really matters for Airflow, but in that case we can probably delete the `start_date` from the `DAG_DEFAULT_ARGS`?

Reply:
Yes, in this case it would always replace the start date within the functions. However, I could _potentially_ see using the `DAG_DEFAULT_ARGS` in full elsewhere, in which case it might be a good idea to keep it there. What do you think?